src/main/resources/data/units/flagshipping.json

```diff
@@ -0,0 +1,1029 @@
+[
+    {
+        "id": "sigma_sol_flagship_1",
+        "baseType": "flagship",
+        "name": "Genesis I",
+		"asyncId": "fs1",
```

Comment:
asyncID is used to know which unit image to place, typically this would be "fs"
We could potentially look at two different images to differentiate the upgraded flagship